src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -2202,47 +2200,18 @@ namespace FanCoilUnits {
 
 			if ( CurDeadBandOrSetback( ZoneNum ) || AirMassFlow < SmallMassFlow ) UnitOn = false;
 
-			// zero the hot & cold water flows
-			mdot = 0.0;
-			SetComponentFlowRate( mdot, FanCoil( FanCoilNum ).ColdControlNode, FanCoil( FanCoilNum ).ColdPlantOutletNode, FanCoil( FanCoilNum ).CWLoopNum, FanCoil( FanCoilNum ).CWLoopSide, FanCoil( FanCoilNum ).CWBranchNum, FanCoil( FanCoilNum ).CWCompNum );
```

Comment:
Lots of extra diffs showing in fancoil - did something get lost in a merge?  Need to double-check this file.